src/code/qualification/generateOutputFolders.m

```diff
@@ -0,0 +1,66 @@
+function  [SectionsOut, Id] = generateOutputFolders(SectionsIn, REOutputFolder)
+%GENERATEOUTPUTFOLDERS Support function: 
+% Explore tree of sections and generate folders of sections within REOutputFolder 
+% linearize the section structure
+%
+%   generateOutputFolders(SectionsIn, REOutputFolder)
+%       SectionsIn (structure or cell) sections to be created
+%       REOutputFolder (string) name of folder where the sections
+%
+
+% Open Systems Pharmacology Suite;  http://open-systems-pharmacology.org
+
+%% ---------------------------------------------------% 
+
+% Check for Sections Class, 
+% if cell: multiple sections, if struct, only one available
+if iscell(SectionsIn)    
+
+% Loop on every section
+for i = 1:length(SectionsIn)
+
+    % Get Section Id
+    Id(i) = SectionsIn{i}.Id;
+    SectionsOut(Id(i)).Id=Id(i);
+    SectionsOut(Id(i)).Title=SectionsIn{i}.Title;
+    
+    % Create folder and add its path in the structure
+    path = fullfile(REOutputFolder, SectionsIn{i}.Title);
+    SectionsOut(Id(i)).path = path;

```

Comment:
what if section title contains characters invalid in directory names (e.g. slash, backslash, *, etc.)? The function will crash then.

E.g. try it with the sections-json below:

```
{
  "Sections": [
    {
      "Id": 1,
      "Title": "Chapter /1",
      "Content": "../Content/Section1.md"
    },
    {
      "Id": 2,
      "Title": "Chapter /2",
      "Content": "../Content/Section2.md",
      "Sections": [
        {
          "Id": 3,
          "Title": "Chapter /2.1",
          "Content": "../Content/Section2.1.md"
        }
      ]
    }
  ]
}
```